api/src/utils/passport-setup.js

```diff
@@ -23,11 +23,20 @@ passport.use(
       clientID: process.env.GOOGLE_CLIENT_ID,
       clientSecret: process.env.GOOGLE_CLIENT_SECRET,
       callbackURL: process.env.OAUTH_CALLBACK_URI,
+      passReqToCallback: true,
     },
-    async (accessToken, refreshToken, profile, cb) => {
+    async (req, accessToken, refreshToken, profile, cb) => {
       // find the user in the database based on their oauth id
       const user = await Member.findOne({ oauthID: profile.id });
 
+      // Set the user's tokens
+      req.session.authedCredentials = {
+        accessToken,
+        refreshToken,
+        clientID: process.env.GOOGLE_CLIENT_ID,
+        clientSecret: process.env.GOOGLE_CLIENT_SECRET,
```

Comment:
I'm a bit confused why the encryption service needs these four pieces of information.

Reply:
Oh is it for this?

> - Refreshes authentication automatically upon expiry while using an active session

Reply:
Yep! Will add a comment to describe this better in the next PR, but the encryption server talks with the Google apis to verify the users OAuth ID matches their encryption key and that the user is who they say they are.